ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/TxStatsUnitTest.java

```diff
@@ -5,18 +5,21 @@
 
 package com.hp.mwtests.ts.arjuna.atomicaction;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-
 import java.io.PrintWriter;
 import java.io.StringWriter;
 
-import org.junit.Test;
-
 import com.arjuna.ats.arjuna.AtomicAction;
 import com.arjuna.ats.arjuna.common.arjPropertyManager;
 import com.arjuna.ats.arjuna.coordinator.TxStats;
+import com.hp.mwtests.ts.arjuna.resources.BasicRecord;
+import org.jboss.byteman.contrib.bmunit.BMScript;
+import org.jboss.byteman.contrib.bmunit.BMUnitConfig;
+import org.jboss.byteman.contrib.bmunit.WithByteman;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 
+@WithByteman
+@BMUnitConfig(debug=true)
```

Comment:
I don't agree with the use of byteman for this since it isn't required. Just begin the transaction, assert the number of inflight transactions and then end it and assert the number of inflight transactions again.